src/css/profile/wearable/changeable/theme-circle/arclistview.less

```diff
@@ -78,7 +78,38 @@
 				margin-top: 0;
 			}
 		}
-		
+
+		&.li-swipe-left {
+			.ui-swipe-icon {
+				position: absolute;
+				width: 98 * @px_base;
```

Comment:
just question.
why use width value 98? and &-background same using width value?

Reply:
I changed this to have 100% of parent.